VideoWeb/VideoWeb/ClientApp/src/app/services/security/auth.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
```

Comment:
@ZJianWang why is this additonal service required? It looks like it's just a sort of proxy for the oidcSecurityService.

Reply:
OK I will try to this without creating a new service

Reply:
looks like need additonal service to share checkAuthCompleted between app.component.ts and auth.guard.tes

Reply:
but why? what does this additional layer provide which you can't achieve with just `oidcSecurityService`?